src/__swaps__/screens/Swap/resources/search/utils.ts

```diff
@@ -1,31 +1,7 @@
 import { ChainId } from '@/chains/types';
 import { SearchAsset } from '@/__swaps__/types/search';
-import {
-  ARBITRUM_ETH_ADDRESS,
-  AVAX_AVALANCHE_ADDRESS,
-  BASE_ETH_ADDRESS,
-  BLAST_ETH_ADDRESS,
-  BNB_BSC_ADDRESS,
-  DEGEN_CHAIN_DEGEN_ADDRESS,
-  ETH_ADDRESS,
-  MATIC_POLYGON_ADDRESS,
-  OPTIMISM_ETH_ADDRESS,
-  ZORA_ETH_ADDRESS,
-} from '@/references';
 import { Address } from 'viem';
-
-const NATIVE_ASSET_UNIQUE_IDS = new Set([
-  `${ETH_ADDRESS}_${ChainId.mainnet}`,
-  `${OPTIMISM_ETH_ADDRESS}_${ChainId.optimism}`,
-  `${ARBITRUM_ETH_ADDRESS}_${ChainId.arbitrum}`,
-  `${BNB_BSC_ADDRESS}_${ChainId.bsc}`,
-  `${MATIC_POLYGON_ADDRESS}_${ChainId.polygon}`,
-  `${BASE_ETH_ADDRESS}_${ChainId.base}`,
-  `${ZORA_ETH_ADDRESS}_${ChainId.zora}`,
-  `${AVAX_AVALANCHE_ADDRESS}_${ChainId.avalanche}`,
-  `${BLAST_ETH_ADDRESS}_${ChainId.blast}`,
-  `${DEGEN_CHAIN_DEGEN_ADDRESS}_${ChainId.degen}`,
-]);
```

Comment:
these were not needed and we can rely on backend driven native assets definitions now.